README.md

```diff
@@ -44,11 +45,19 @@ Sign in is currently takes any email or password. It just does some front end st
 
 Right now there is no button to take you to the user edit page so you will have to manually change the url to http://localhost:3000/family/edit
```

Comment:
Not sure what this comment refers to. Family edit is somewhat out of scope at the moment. Work is focused on Pre-Registration MVP that has a households model.

Reply:
@davidpickering That comment is already there in the master branch. I have just added the comments on Google API integration along with what has been there already. If that comment was put there purposefully, changing that in my PR and merging it could remove it from the master.